src/main/java/reposense/system/LogsManager.java

```diff
@@ -0,0 +1,99 @@
+package reposense.system;
+
+import java.io.IOException;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+/**
+ * Configures and manages the loggers and handlers, including their levels
+ */
+public class LogsManager {
+    private static final int FILE_COUNT = 2;
+    private static final int MEGABYTE = (int) (Math.pow(2.0, 20));
```

Comment:
Don't think we  should be using `Math.pow` which returns a `double` and can have floating point errors, also when we cast this back to `int`. Can try using the absolute value, or use `1 << 20`